dc1/foreground_sim/setup_sim.py

```diff
@@ -35,45 +28,32 @@
         #    print(f"Output file exists: {fname_out}")
         #    continue
         fname_in = (
-            f"/global/cfs/cdirs/cmbs4/dm/mbs/"
-            f"202211_LAT_fg_cmb/4096/combined_foregrounds_cmb_fixdip{suffix}/0000/"
-            f"cmbs4_combined_foregrounds_cmb_fixdip{suffix}_uKCMB_LAT-{band}_nside4096_0000.fits"
+            f"/global/cfs/cdirs/cmbs4/dc/dc0/sky/4096/combined_foregrounds{suffix}/0000/cmbs4_combined_foregrounds{suffix}_uKCMB_LAT-{band}_nside4096_0000.fits"
         )
         if not os.path.isfile(fname_in):
             raise RuntimeError(f"Input file does not exist: {fname_in}")
         print(f"Reading {fname_in}")
-        m = read_map(
+        fg = hp.read_map(
             fname_in,
             None,
             nest=True,
             verbose=False,
             dtype=np.float32,
         )
         if not os.path.isfile(fname_out):
-            write_healpix(fname_out, m * 1e-6, coord="C", nest=True)
+            write_healpix(fname_out, fg * 1e-6, coord="C", nest=True)
```

Comment:
can you also add the unit to the output files? In `hp.write_map` is `column_units`

Reply:
Sadly no, the HDF5 format does not recognize that keyword right now. Easy to fix but we don't want to change the revision of TOAST used to run the sims.